docs/topics_5/web-adding-global-custom-labels.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+// * docs/web-console-guide_5/master.adoc
+[id='adding_global_custom_labels_{context}']
+= Adding global custom labels
+
+The {ProductShortName} includes a preconfigured set of global labels, which apply to all projects.
+
+You can define your own custom global labels.
+
+.Procedure
+
+. In the {WebName}, click *Labels configuration*.
++
+image::web-labels-config_51.png[Labels configuration screen]
```

Comment:
Same problem. Image does not show enough of UI or the custom labels tab. Take it out.